sample_app/sampleapp_common.c

```diff
@@ -373,6 +380,35 @@ static bool app_subslot_is_output (app_subslot_t * p_subslot)
    }
 }
 
+/********************************* Set outputs ********************************/
+
+/* Set LED state.
+ *
+ * Compares new state with previous state to avoid disk operations.
+ *
+ * @param led_state        In:    New LED state
+ * @param verbosity        In:    Verbosity
+ */
+static void app_handle_data_led_state (bool led_state, int verbosity)
+{
+   static bool previous_led_state = false;
+
+   if (led_state != previous_led_state)
+   {
+      app_set_led (APP_DATA_LED_ID, led_state, verbosity);
+   }
+   previous_led_state = led_state;
+}
+
+/* Set outputs to default value.
+ *
+ * @param verbosity        In:    Verbosity
+ */
+static void app_set_outputs_default_value (int verbosity)
```

Comment:
Suggestion:
Add printout printf ("Setting outputs to default values") in this function and remove printouts in calling functions.

Reply:
Done